cep-??.md

```diff
@@ -0,0 +1,131 @@
+<table>
+<tr><td> Title </td><td> Environment yaml specification </td>
+<tr><td> Status </td><td> Draft </td></tr>
+<!-- <tr><td> Status </td><td> Draft | Proposed | Accepted | Rejected | Deferred | Implemented </td></tr> -->
+<tr><td> Author(s) </td>
+    <td>
+      Eric Dill
+      Marius van Niekerk
+      Jaime Rodríguez-Guerra
```

Comment:
@jaimergp you okay being a co-author of this?

Reply:
Yes, thank you!